backend/src/api/helpers.ts

```diff
@@ -75,3 +77,35 @@ export const Unauthorized: APIGatewayProxyResult = {
   statusCode: 403,
   body: ''
 };
+
+export const sendEmail = async (
```

Comment:
Can we use something like [nodemailer](https://nodemailer.com/about/) instead? We can use https://nodemailer.com/transports/ses/

The main benefits of this is that 1) it's easy to swap in SES for another mail solution or custom SMTP server, and 2) more importantly, it's an easier API to use to add more advanced features, such as a HTML body or adding attachments.

With the AWS SES SDK, you have to call [SendRawEmail](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/send-email-raw.html) and then manually construct the message using MIME (not very fun), so nodemailer seems to be an easier solution for this.

Reply:
Yep, I'll switch this out